pkg/profiler/cpu/cpu.go

```diff
@@ -1174,3 +1175,14 @@ func preprocessRawData(rawData map[profileKey]map[combinedStack]uint64) profile.
 
 	return res
 }
+
+func getArch() elf.Machine {
+	switch runtime.GOARCH {
```

Comment:
Why does Go loves strings so much 😢

Reply:
ikr -_-
I actually care about this one so maybe time to just make a PR to some libraries 😛